webapp/channels/src/components/drafts/draft_title/draft_title.test.tsx

```diff
@@ -10,31 +10,36 @@ import type {UserProfile} from '@mattermost/types/users';
 
 import mockStore from 'tests/test_store';
 import Constants from 'utils/constants';
+import {wrapIntl} from 'utils/test_intl';
 
 import DraftTitle from './draft_title';
 
 describe('components/drafts/draft_actions', () => {
+    const channel = {
+        type: 'O',
+        display_name: 'Test Channel',
+    } as Channel;
+
     const baseProps = {
-        channelType: '' as Channel['type'],
-        channelName: '',
+        channel,
         membersCount: 5,
         selfDraft: false,
         teammate: {} as UserProfile,
         teammateId: '',
-        type: '' as 'channel' | 'thread',
+        type: 'channel' as 'channel' | 'thread',
     };
 
     it('should match snapshot', () => {
         const store = mockStore();
 
-        const wrapper = shallow(
+        const {container} = render(wrapIntl(
```

Comment:
you dont need to introduce another util if you use renderWithContext directly.

Reply:
Ah thanks! Done.